File: YukariLauncher/src/main/java/net/kdt/pojavlaunch/authenticator/listener/ErrorListener.java
package net.kdt.pojavlaunch.authenticator.listener;

/**
 * 完全な失敗が発生したときに呼び出されます。UIスレッドで実行されることが保証されています。
 */
public interface ErrorListener {
    /**
     * ログインエラーが発生したときに呼び出されます。
     * @param errorMessage エラーの詳細
     */
    void onLoginError(Throwable errorMessage);
}

File: YukariLauncher/src/main/java/com/arata/yukarilauncher/feature/login/ErrorListener.kt
package com.arata.yukarilauncher.feature.login

/** ログインエラーリスナー。 */
interface ErrorListener {
    /** ログインエラー時に呼び出される。 @param errorMessage エラー内容 */
    fun onLoginError(errorMessage: Throwable)
}